### PR TITLE
Add queries, data and schema related with join

### PR DESCRIPTION
**Pull request by Pi1998**
# Branch Changes 💯 

- Used primary-key & foreign-key mechanisms for joining tables 👍 
- Queried multiple tables using Join 🔢 

****Here are the screenshots for respective requirements****

- *Species and Owners Tables*

Screenshot 2023-11-04 112854
Screenshot 2023-11-04 112926
Screenshot 2023-11-04 112944

-*Including Species Ids in animal table*

Screenshot 2023-11-04 113007

-*Modifying animal table*

Screenshot 2023-11-04 113056
Screenshot 2023-11-04 113113
Screenshot 2023-11-04 152122

-*Here are the requested queries using `Join`*

Screenshot 2023-11-04 153659
Screenshot 2023-11-04 162317
Screenshot 2023-11-04 162954
Screenshot 2023-11-04 163245
Screenshot 2023-11-04 163849
Screenshot 2023-11-04 183130
Screenshot 2023-11-04 165208

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -87,3 +87,50 @@ Where date_of_birth Between '1990-1-1' and '2000-1-1'
...
+From owners o
+Join animals a On o.id = a.owner_id
+Join species s On a.species_id = s.id
+Where o.full_name = 'Jennifer Orwell' And s.name = 'Digimon';
+
+SELECT o.full_name As owner, 
+       COALESCE(a.name, 'They all tried to escape :3') As pet_who_loves_home
+From owners o
+Left Join animals a On o.id = a.owner_id And a.escape_attempts = 0
+Where o.full_name = 'Dean Winchester';
```
Comment (nahnahdev) [inline queries.sql]: - [ ] Great job so far @Pi1998 There are a few changes I will suggest you implement and you'll be on your way to the next Milestone :+1: 
- [ ] I noticed that your results for this query `List all animals owned by Dean Winchester that haven't tried to escape.` is different from the expected result. So I will suggest you rewrite the query and ensure that it is returning the right response. :+1: 
- [ ] From the data you inserted, you can see that none of the animals owned by `Dean Winchester` have tried to escape. As a result, your query should return 0 or empty.

File: queries.sql [hunk 2]
```diff
@@ -87,3 +87,50 @@ Where date_of_birth Between '1990-1-1' and '2000-1-1'
...
+Where o.full_name = 'Melody Pond';
+
+Select a.name, s.name
+From species s
+Join animals a On s.id = a.species_id
+Where s.name = 'Pokemon';
+
+Select o.full_name as owner, a.name as their_pet
+From owners o
+Left Join animals a On o.id = a.owner_id;
```
Comment (nahnahdev) [inline queries.sql:111]: - [ ] Also, this query `List all owners and their animals, remember to include those that don't own any animal.` needs to return 11 rows in total. But yours is currently returning only 8 so kindly fix that as well :+1:

**Review by nahnahdev — CHANGES_REQUESTED** [review]
## Project Status: CHANGES REQUIRED :recycle: 
Hi @Pi1998  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :fire: 
- All required files present :heavy_check_mark: 
- Each file contains its corresponding data :heavy_check_mark: 
- Most queries return the right results :heavy_check_mark: 
- Professional commit messages :heavy_check_mark: 
- Most project requirements implemented :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 2 above)
Comment (Pi1998) [reply queries.sql:111]: > * [ ]  Also, this query `List all owners and their animals, remember to include those that don't own any animal.` needs to return 11 rows in total. But yours is currently returning only 8 so kindly fix that as well 👍

This is the result of the query. Can you show me other way???
Screenshot 2023-11-04 182308

**Review by TadesseDev — APPROVED** [review]
### Hi @Pi1998 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @Pi1998 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
AmazingCreepyFrogmouth-max-1mb




## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**